n26/core/templates/cotton/n26/owned_lines.html

```
@@ -10,7 +10,8 @@
 Which acts a copy offers, and which of them is red, is the structure's word and
 not this template's (see n26/includes/row_action.html), so an act added there
 appears here with nothing edited. A part offers no move, because
-Operation.move refuses a firing line.
+Operation.move refuses a firing line. The Sell and more-menu pair is
+<c-n26.owned-actions>, which the model's own card folds into one menu.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)